docs/integrations/create-graphiql-plugin.md

```diff
@@ -0,0 +1,142 @@
+# Graphiql custom plugin
+
+You can easily create a graphiql plugin and integrate it in mercurius graphiql instance.
+
+[More info here.](https://github.com/graphql/graphiql)
+
+[GraphiQL Explorer Plugin example.](https://github.com/graphql/graphiql/tree/main/packages/graphiql-plugin-explorer)
+
+## Create the plugin
+
+### Plugin component
+
+A Graphiql plugin is an object that exports these values:
+
+* title: string. The title of the plugin
+* icon: React component. The icon shown in the toolbar
+* content: React component with the plugin implementation
+ 
+It can be created using the sample:
+
+```javascript
+import React from 'react'
+
+function Content() {
+  return (
+    <div style={{ maxWidth: '300px' }}>
+      <div style={{ height: '100%' }}>This is a sample plugin</div>
+    </div>
+  )
+}
+
+function Icon() {
+  return <p>GE</p>
+}
+
+export function parseFetchResponse(data) {
+  if (data.data) {
+    // Do something with the data returned by the fetch
+  }
+  return data
+}
```

Comment:
it's not really clear to me what the purpose of this function is. why would anyone want to modify the response of the url at which the plugin lives? even assuming that one wants to, can't we just provide a default implementation which does something sensible?

Reply:
note this is also somewhat confusing because the fetch API has no data property in the response

Reply:
I need to explain better why this function is required.